### PR TITLE
Update CoreFx script

### PR DESCRIPTION
**Pull request by ryanbrandenburg**
We need to make some changes to this script to make it work on the CI. Namely, we must allow the use of a GitHub access token.

### CODE REVIEW

File: scripts/common.psm1 [hunk 1]
```diff
@@ -143,9 +143,10 @@ function CommitUpdatedVersions([hashtable]$updatedVars, [xml]$dependencies, [str
         $remote = "origin"
         $baseBranch = "dev"
 
+        Invoke-Block { & git stash }
```
Comment (natemcmaster) [inline scripts/common.psm1]: What's this about?

Comment (ryanbrandenburg) [reply scripts/common.psm1]: We've already made changes to build/dependencies.props, if we don't stash our changes before we checkout checkout will fail.

**Comment by pakrym** [comment]
https://github.com/aspnet/Universe/issues/1063

File: scripts/common.psm1 (hunk 1 above)
Comment (natemcmaster) [reply scripts/common.psm1]: Is there any reason we cannot but the branch checkout to the top of the script? stash pop is prone to merge conflicts.

File: scripts/UpdateDependencies.ps1 [hunk 2]
```diff
@@ -15,6 +15,12 @@ Import-Module -Scope Local -Force "$PSScriptRoot/common.psm1"
 Set-StrictMode -Version 1
 [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
 
+$GitHubEmail = $ConfigVars["GithubEmail"]
+$GitHubUsername = $ConfigVars["GithubUsername"]
+$GitHubPassword = $ConfigVars["GithubToken"]
+
+Set-GitHubInfo $GitHubPassword $GitHubUsername $GitHubEmail
```
Comment (natemcmaster) [inline scripts/UpdateDependencies.ps1]: So, if I want to run this locally without creating a PR, how would I skip this? Might be useful to add a `-NoPush` and/or `-NoCommit` switch to skip this

**Comment by ryanbrandenburg** [comment]
:up::date:. @pakrym this should fix #1063.

**Pull request merged**